tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchCMISTests.java

```diff
@@ -104,6 +104,22 @@ public void basicCMISQuery()
         searchQueryService.expectSomeResultsFromQuery(query, user1);
     }
 
+    @TestRail (description = "Verify that we can return a list of fields from a document via a CMIS query against Elasticsearch.", section = TestGroup.SEARCH, executionType = ExecutionType.REGRESSION)
+    @Test (groups = TestGroup.SEARCH)
+    public void returnFieldListFromDoc()
+    {
+        SearchRequest query = req("cmis", "SELECT cmis:name, cmis:objectId, cmis:lastModifiedBy, cmis:creationDate, cmis:contentStreamFileName FROM cmis:document WHERE CONTAINS('*')");
```

Comment:
Does the list of field name affect the results ?

If yes, can we check ?

If no, then i assume there must be a negative test - eg. fields that don't exist and cause semantic error ?

Reply:
Note: with Solr, I think I can see 400 in proxy as expected, eg.

`proxy_1                      | 86.15.70.161 - admin [13/Apr/2022:16:36:02 +0000] "POST /alfresco/api/-default-/public/cmis/versions/1.1/browser HTTP/1.1" 400 96 "-" "Apache-Chemistry-OpenCMIS/1.1.0 (Java 11.0.10; Mac OS X 12.3.1)"`

On client-side (using CMIS Workbench), I see:

```
CMIS Exception: invalidArgument Invalid column for cmis:document cmis:nam
  Error Content: {"exception":"invalidArgument","message":"Invalid column for cmis:document cmis:nam"}
```